bundles/AdminBundle/Controller/Admin/TranslationController.php

```diff
@@ -899,9 +899,11 @@ public function xliffImportElementAction(Request $request, ImportDataExtractorIn
             $importerService->import($attributeSet);
         } catch (\Exception $e) {
             Logger::err($e->getMessage());
+            $success = $e instanceof Element\Exception\ElementNotFoundException ?? false;
```

Comment:
I doubt that the result of `instanceof` can ever be `null`.

Reply:
true!!....changed it 😉 thanks!